WORKSPACE

```diff
@@ -96,6 +102,9 @@ go_repository(
 
 go_repository(
     name = "com_github_urfave_cli",
+    build_extra_args = ["-exclude=vendor"],
```

Comment:
Why was this needed, the underlying commit we are pulling shouldn't have changed at all? Same question for the following two lines as well.

Reply:
this change was related to something i had on my local environment.

a clean pull removes the need for these lines.

patched